samples/excel/42-range/range-cell-control.yaml

```diff
@@ -0,0 +1,120 @@
+order: 23
+id: excel-cell-controls
+name: Checkboxes
+description: This sample shows how to add and remove checkboxes from a table.
+host: EXCEL
+api_set:
+    ExcelApi: '1.18'
+script:
+    content: |-
+        document.getElementById("setup").addEventListener("click", () => tryCatch(setup));
+        document.getElementById("add-checkboxes").addEventListener("click", () => tryCatch(addCheckboxes));
+        document.getElementById("remove-checkboxes").addEventListener("click", () => tryCatch(removeCheckboxes));
+
+        // Add checkboxes to the table.
+        async function addCheckboxes() {
+          await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            let range = sheet.getRange("FruitTable");
```

Comment:
The code will add the checkbox control (formatting) to the entire range for the table. I doubt that is what we want to demonstrate?

Reply:
So why didn't the whole table display checkboxes then?

Reply:
Because the cells in the table header and first column have string values. If you change that to TRUE or FALSE you will see a checkbox show up.

Reply:
That is exactly what I'm trying to demonstrate. All boolean values get changed to checkboxes in the selected range.

Reply:
I updated the snippet to get the Analysis column: 

`const range = sheet.getRange("FruitTable").getColumn(1)`

Reply:
Excellent! That is a lot better.